fleet/backend/backend.go

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package backend
+
+import "github.com/ns1labs/orb/pkg/types"
+
+type Backend interface {
+	Handlers() (types.Metadata, error)
```

Comment:
Handlers and Inputs are not a generic Backend concept, they are pktvisor specific